test-fixture/src/sim/http3_connection.rs

```diff
@@ -303,15 +303,15 @@ impl Requests {
         loop {
             let remaining = self.remaining.get_mut(&stream_id).unwrap();
             let end = min(*remaining, DATA.len());
-            let sent = c.send_data(stream_id, &DATA[..end]).unwrap();
+            let sent = c.send_data(stream_id, &DATA[..end], now()).unwrap();
```

Comment:
All these added now parameters is a bit of a distraction from the core reason for the change.  Could that be split out?

Reply:
I'll break those out into another PR.